### DISCUSSION TITLE
export fs::asyncify publically?

### DISCUSSION THREAD

**Discussion by cehteh** [tokio-rs/tokio#3687] [Ideas] [answered]
asyncify is a small tool one sometimes needs. I could just copypaste and implement it where needed, its just a few lines. But perhaps its a good choice to bless it as public interface (from runtime then?). Make the Result generic and done. Or did I overseen something that would make this infeasible?

**Comment by Darksonn** [comment, marked as answer]
It's just a wrapper around `tokio::task::spawn_blocking` that converts panics in the spawned task into an io error. Use `spawn_blocking` instead.
